share/cpt/cork.cpt

```diff
@@ -141,7 +141,8 @@
 -0.019608	229/237/238	-0.011764	230/238/238
 -0.011764	230/238/238	-0.003922	230/238/237
 -0.003922	230/238/237	0.003922	231/239/237
```

Comment:
```suggestion
```

Reply:
This change is incorrect. There are duplicated records for `-0.003922` to `0.003922`.